aptos-move/framework/aptos-framework/sources/configs/Version.move

```diff
@@ -32,17 +31,18 @@ module AptosFramework::Version {
     }
 
     /// Updates the major version to a larger version.
+    /// This is only used in test environments and outside of them, the core resources account shouldn't exist.
     public(script) fun set_version(account: signer, major: u64) acquires Version {
         SystemAddresses::assert_core_resource(&account);
```

Comment:
@davidiw @zekun000 I wasn't able to switch to 0x1 here because of some tests depending on this set_version to trigger reconfiguration. Just to confirm though, this should not be used at all outside of tests right?

Reply:
This seems to be something we'd want to only enable in our governance platform. I guess we're not quite there yet, so ... it is what it is.